accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts

```diff
@@ -1616,22 +1616,21 @@ export class RPTUtil {
      * @memberOf RPTUtil
      */
     public static getLabelForElementHidden(element: Element, ignoreHidden) {
-
         // Check if the global RPTUtil_LABELS hash is available, as this will contain the label nodes based on
         // for attribute.
-        if (!RPTUtil.getCache(element.ownerDocument,"RPTUtil_LABELS", null)) {
+        //if (!RPTUtil.getCache(element.ownerDocument,"RPTUtil_LABELS", null)) {
+        let root = element.getRootNode();
+        if (!RPTUtil.setCache((root.nodeType === 11)? <ShadowRoot>root : <Document>root, "RPTUtil_LABELS", null)) {
```

Comment:
I think you meant `getCache` here instead of `setCache`

Reply:
cleanup error. fixed.